src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,189 @@
+    box-shadow: inset 0 2px 0 0 var(--pf-c-tabs__item--current--Color);
+  }
+
+  // Hover state
+  &:not(.pf-m-start-current) .pf-c-tabs__scroll-button:nth-of-type(1):hover,
+  &:not(.pf-m-end-current) .pf-c-tabs__scroll-button:nth-of-type(2):hover {
+    border-top-color: var(--pf-c-tabs__item--hover--BoxShadowColor);
+    box-shadow: inset 0 2px 0 0 var(--pf-c-tabs__item--hover--BoxShadowColor);
+  }
+}
+
+// Tab List
+.pf-c-tabs__list {
+  display: flex;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+}
+
+// Tab Items
+.pf-c-tabs__item {
+  flex: none;
+  border-top: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
+  border-bottom: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
+  border-left: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
+
+  &:last-child {
+    border-right: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
+  }
+
+  &.pf-m-current {
+    z-index: 1;
+    border-top-color: var(--pf-c-tabs__item--current--Color);
+    border-bottom: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BackgroundColor);
+
+    .pf-c-tabs__button {
+      color: var(--pf-c-tabs__item--current--Color);
+      box-shadow: inset 0 2px 0 0 var(--pf-c-tabs__item--current--Color);
+    }
+  }
+
+  // Item Hover state
+  &:not(.pf-m-current):hover {
+    border-top-color: var(--pf-c-tabs__item--hover--BoxShadowColor);
+  }
+
+  // Button Hover state
+  &:not(.pf-m-current):hover .pf-c-tabs__button {
+    box-shadow: inset 0 2px 0 0 var(--pf-c-tabs__item--hover--BoxShadowColor);
+  }
+}
+
+// Secondary tabs
+.pf-c-tabs.pf-m-tabs-secondary {
+  .pf-c-tabs__item {
+    z-index: initial;
+    border-top-color: var(--pf-c-tabs__item--BorderColor);
+    border-right: none;
+    border-bottom: none;
+    border-left: none;
+
+    // Hover state
+    &:not(.pf-m-current):hover .pf-c-tabs__button {
+      color: var(--pf-c-tabs__item--hover--Color);
+    }
+  }
+
+  // Different styling for secondary tab buttons
+  .pf-c-tabs__button {
+    box-shadow: initial;
+    box-shadow: none;
+
+    &:hover {
+      box-shadow: none;
+    }
+  }
+}
+
+// Tab buttons
+.pf-c-tabs__button {
+  padding-top: var(--pf-c-tabs__button--PaddingTop);
+  padding-right: var(--pf-c-tabs__button--PaddingRight);
+  padding-bottom: var(--pf-c-tabs__button--PaddingBottom);
+  padding-left: var(--pf-c-tabs__button--PaddingLeft);
+  user-select: none;
+  border: initial;
+}
+
+// Directional scroll buttons
+.pf-c-tabs__scroll-button {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  z-index: 2;
+  display: none;
+  flex: none;
+  width: var(--pf-c-tabs__scroll-button--Width);
+  line-height: 1;
+  background-color: var(--pf-c-tabs__scroll-button--BackgroundColor);
+  border: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__scroll-button--BorderColor);
+  border-radius: initial;
+
```

Comment:
if you set the color here the theming should work.

Reply:
right so because you aren't specifying a colour for the icon it's just taking the default and when the theme is toggled anything dark goes to light. So you'd have to set the colour of the explicitly here to override that for the theme. Does that help?

Reply:
Confusion is gone and `--pf-global--BackgroundColor--100` now adjusts to dark themes. Thanks for figuring that out! 👍 

Cards, Nav, and Popover are the only other components using `--pf-global--BackgroundColor--100` that could be affected when the fix included in this PR is merged.

- Card has custom styling within `placeholders.scss` that switches its background to `--pf-global--BackgroundColor--dark-transparent-200` while in dark mode. It probably shouldn't be doing that in there, but that's its own problem.
- Nav sets a background color variable, but then never actually uses it, so that's fine.
- Popover had the same issue I did with this Tabs component and it now switches to a dark background, but it's barely visible and should probably switch to using the same `--pf-global--BackgroundColor--dark-transparent-200` that Card does to look okay.